### DISCUSSION TITLE
Pagination problem of member list in base

### DISCUSSION THREAD

**Discussion by c2006878** [nocodb/nocodb#9416] [Q&A]
I am adding pagination to the member list in each base. But why can't the backend receive the query passed by the front end? Does anyone know why?
![image](https://github.com/user-attachments/assets/1bb448e2-51d1-4fe2-9bf7-f26cdeb1ff64)

![image](https://github.com/user-attachments/assets/879033e6-68c3-41c6-aa82-3832a2a2e58f)

**Comment by pranavxc** [comment]
Pagination has to be implemented at the `BaseUsersService` and `BaseUser` level as well. At the moment it's fetching all the users, to achieve pagination you have to update these 2 files as well, `BaseUsersService` and `BaseUser` .
